### DISCUSSION TITLE
Verification of academic email

### DISCUSSION THREAD

**Discussion by Junbum-Cho** [RosettaCommons/rosetta#202] [ROSIE] [answered]
Hello, 

I am a student at Avon Old Farms School in US Connecticut, and I am hoping to use Rosetta or Rosie for academic purposes. I would like to confirm my email can be used as an academic email: choj@avonoldfarms.com 

Thank you:)

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know, - this should be fixed now! Please feel free to re-open this issue if you still have trouble login-in.
